app/javascript/pages/IndividualEvents/index.js

```diff
@@ -261,86 +262,172 @@ const DestroyDialog = ({ onDelete, onCancel, popover }) => (
+      accessor: d => d.duration,
+    },
+    {
+      id: 'type',
+      Header: 'Type',
+      accessor: d => d.eventType.title,
+    },
+    {
+      id: 'approval',
+      Header: 'Approval',
+      accessor: d => eventStatusIcon(d),
+      width: 75,
+      style: { textAlign: 'center' },
+    },
+    {
+      id: 'actions',
+      Header: 'Actions',
+      accessor: d => d,
+      sortable: false,
+      Cell: props => (
+        <span className="s.actionColumn">
+          <button
+            className={`${s.btn} ${s.confirmBtn} ${s.leftAligned}`}
+            onClick={() => togglePopover('editIndividualEvent', props.value)}
+          >
+            Edit
+          </button>
+          <button
+            className={`${s.btn} ${s.deleteBtn}`}
+            onClick={() => togglePopover('destroyIndividualEvent', props.value)}
+          >
+            Delete
+          </button>
+        </span>
+      ),
+    },
+  ]
+
+  return (
+    <div className={s.eventsTable}>
+      <div className={s.personalHeader}>
+        <div className={s.actionBar}>
+          <button className={s.createAction} onClick={() => togglePopover('editIndividualEvent')}>
+            Add Event
+          </button>
+        </div>
+        <h1>Individual Events</h1>
+      </div>
+      <ReactTable
+        NoDataComponent={() => null}
+        data={currentUser.individualEvents}
+        columns={individualEventsColumns}
+        showPagination={false}
+        pageSize={currentUser.individualEvents.length}
+        defaultSorted={[{ id: 'date', desc: true }]}
+        minRows={0}
+      />
+      {popover && popover.type === 'editIndividualEvent' ? (
+        <CreateEditDialog
+          offices={offices}
+          eventTypes={eventTypes}
+          organizations={organizations}
+          popover={popover}
+          onSubmit={handleSubmit(createEditIndividualEvent)}
+          onCancel={() => togglePopover('editIndividualEvent')}
+        />
+      ) : null}
+      {popover && popover.type === 'destroyIndividualEvent' ? (
+        <DestroyDialog
+          popover={popover}
+          onDelete={() => deleteIndividualEvent(popover.data.id) && togglePopover('destroyIndividualEvent')}
+          onCancel={() => togglePopover('destroyIndividualEvent')}
+        />
+      ) : null}
+    </div>
+  )
+}
+
+const organizedEventsColumns = [
+  {
+    id: 'event',
+    Header: 'Event',
+    accessor: d => d.title,
+  },
+  {
+    id: 'organization',
+    Header: 'Organization',
+    accessor: d => d.organization.name,
+  },
+  {
+    id: 'date',
+    Header: 'Date',
+    Cell: props => <span>{moment(props.value).format('MMMM D, YYYY')}</span>,
+    accessor: d => d.startsAt,
+  },
+  {
+    id: 'duration',
+    Header: 'Duration (min)',
+    accessor: d => d.duration,
+  },
+  {
+    id: 'type',
+    Header: 'Type',
+    accessor: d => d.eventType.title,
+  },
+  {
+    id: 'location',
+    Header: 'Location',
+    accessor: d => d.location,
```

Comment:
I believe
```suggestion
    accessor: 'location',
```
works, too. A bit shorter.

Reply:
ah yes thanks for that!